success-criteria.md

```diff
@@ -374,13 +348,17 @@ stakeholder group.
 
     -   Antitrust intervention
 
+- Freedom to browse anonymously at will and have that choice respected by all entities. The ability to access content both freely and anonymously is essential to allowing ideas to fairly propagate.
+
 #### Interests of Individual People
 
 -   Same interests as society-level plus:
 
     -   Fast, frictionless and secure access to a wide array of internet-enabled
         content and services that make the Web so valuable
 
+-   - A right to choose private browsing without systems attempting to remove that anonymity.
+
```

Comment:
A right to object is not a meaningful right. A right to stop is a meaningful right. Article 12 of the Universal Declaration of Human Rights states privacy as a human right. Propose to move, strike and replace the 'right to object' section.